apps/browser-extension-wallet/src/views/browser-view/features/assets/components/AssetsPortfolio/AssetsPortfolio.tsx

```diff
@@ -35,6 +38,16 @@ export interface AssetsPortfolioProps {
   isLoadingFirstTime?: boolean;
 }
 
+const searchTokens = (data: IAssetDetails[] | IRow[], searchValue: string) => {
+  const fields = ['name', 'policyId', 'fingerprint']; // Fields to search
+  const lowerSearchValue = searchValue.toLowerCase();
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  return data.filter((item: any) =>
```

Comment:
How much work to replace this any with a proper type? is not here IRow?

Reply:
Why not just remove it? TS will infer the type.
```suggestion
  return data.filter((item) =>
```

Reply:
Fixed on 55a094f